packages/serverless-workflow-language-service/tests/SwfJsonLanguageService.test.ts

```diff
@@ -697,7 +721,338 @@ describe("SWF LS JSON", () => {
+          const content = `{
+            "dataInputSchema": "path/to/schema",
+            "functions": [
+                {
+                    "name": "testFunc1",
+                    "type": "asyncapi",
+                    "operation": "http://path_to_remote_asyncApiFile/"
+                },
+                {
+                    "name": "testFunc2",
+                    "type": "rest",
+                    "operation": "http://path_to_remote_openApiFile/"
+                },
+                {
+                    "name": "expressionFunc1",
+                    "type": "expression",
+                    "operation": "."
+                },
+                {
+                    "name": "expressionFunc2",
+                    "type": "expression",
+                    "operation": "."
+                },
+                {
+                  "name": "expressionFunc3",
+                  "type": "expression",
+                  "operation": ${nodeValue}
+              }
+            ]
+        }` as ContentWithCursor;
+          const { completionItems } = await codeCompletionTester(
+            getJsonLsForJqExpressionTests(),
+            documentUri,
+            content,
+            false
+          );
+          expect(completionItems.length).toMatchSnapshot();
+          if (completionItems.length > 10) {
+            expect(completionItems.slice(-5)).toMatchSnapshot();
+          } else {
+            expect(completionItems).toMatchSnapshot();
+          }
+        }
+      );
+    });
+    describe.each([
+      ["Jq data condition completion for built-in functions"],
+      ["Jq data condition completion for reusable functions"],
+      ["Jq data condition completion for variables"],
+    ])(`%s completion`, () => {
+      test.each([...getJqBuiltInFunctionTests(), ...getJqReusableFunctionTests(), ...getJqVariableTests()])(
+        "%s",
+        async (_description, nodeValue) => {
+          const content = `{
+        "dataInputSchema": "/path/to/json_schema.json",
+        "functions":[
+          {
+            "name": "testFunc",
+            "type": "rest",
+            "operation":"http://path_to_remote_asyncApiFile/"
+          }
+        ],
+        "states": [
+        {
+          "name": "testState",
+          "type": "switch",
+          "dataConditions": [
+            {
+              "condition": ${nodeValue},
+              "transition": "Scale up on Ansible"
+            }
+          ],
+        }
+      ]
+    }` as ContentWithCursor;
+          const { completionItems } = await codeCompletionTester(
+            getJsonLsForJqExpressionTests(),
+            documentUri,
+            content,
+            false
+          );
+          expect(completionItems.length).toMatchSnapshot();
+          if (completionItems.length > 10) {
+            expect(completionItems.slice(-5)).toMatchSnapshot();
+          } else {
+            expect(completionItems).toMatchSnapshot();
+          }
```

Comment:
This should be enough to test only the first 5 elements.
This can be applied to other Jq tests.

```suggestion
          expect(completionItems.slice(0,5)).toMatchSnapshot();
```